src/site/twitch.tv/modules/hidden-elements/HiddenElementsModule.vue

```diff
@@ -196,12 +196,25 @@ export const config = [
 	button[data-test-selector="unfollow-button"] {
 		display: none !important;
 	}
+	// Prevents "React" button from overlapping the notification / subscribe button
+	div[data-target="channel-header-right"] > div:first-child {
+		margin-right: -1rem !important;
+	}
 }
 
 .seventv-hide-live-notification-button {
 	button[data-a-target="notifications-toggle"] {
 		display: none !important;
 	}
+	// Remove entire div containing live noti button to get rid of left padding
+	div[data-target="channel-header-right"]
+		> *:nth-child(2)
+		> *:nth-child(1)
+		> *:nth-child(2)
+		> *:nth-child(1)
+		> *:nth-child(2) {
+		display: none !important;
+	}
```

Comment:
This is likely to break, at this point it may be preferable to hook the react component itself rather than use purely CSS.